test/cluster/lwt/test_lwt_with_counters_during_tablets_resize_and_migrations.py

```diff
@@ -0,0 +1,419 @@
+        )
+        src_host_id, src_shard = random.choice(replicas)
+        src_server = host_id_to_server.get(src_host_id)
+        if src_server is None:
+            logger.info(
+                "Source host_id %s for token %s not found in host_id_to_server, re-pick (attempt %d)",
+                src_host_id,
+                token,
+                attempt,
+            )
+            continue
+
+        if random.random() < intranode_ratio:
+            dst_host_id = src_host_id
+            dst_server = src_server
+            dst_shard = 0 if src_shard != 0 else 1
+        else:
+            replica_hids = {h for (h, _sh) in replicas}
+            src_rack = server_id_to_rack[src_server.server_id]
+
+            same_rack_candidates = [
+                s for s in servers if server_id_to_rack[s.server_id] == src_rack
+                and server_id_to_host_id[s.server_id] not in replica_hids
+            ]
+
+            if not same_rack_candidates:
+                logger.info(
+                    "No same-rack non-replica candidate for token %s, re-pick (attempt %d)",
+                    token,
+                    attempt,
+                )
+                continue
+
+            dst_server = random.choice(same_rack_candidates)
+            dst_host_id = server_id_to_host_id[dst_server.server_id]
+            dst_shard = 0
+
+        try:
+            await _move_tablet_with_retry(
+                manager, src_server, tester.ks, tester.tbl,
+                src_host_id, src_shard, dst_host_id, dst_shard, token,
+                timeout_s=60
+            )
+            await _move_tablet_with_retry(
+                manager, src_server, tester.ks, counter_table,
+                src_host_id, src_shard, dst_host_id, dst_shard, token,
+                timeout_s=60
+            )
+
+            migration_count += 1
+            logger.info(
+                "Completed migration #%d (token=%s -> %s:%d) for %s.%s and %s.%s",
+                migration_count, token, dst_server.ip_addr, dst_shard,
+                tester.ks, tester.tbl, tester.ks, counter_table,
+            )
+            await asyncio.sleep(random.uniform(*pause_range))
+            continue
+        except Exception as e:
+            if _is_tablet_in_transition_http_error(e):
+                logger.info("Token %s in transition, switching token (attempt %d)",
+                            token, attempt)
+                continue
+            if _is_no_replica_on_src_error(e) or _is_dst_already_replica_error(e):
+                logger.info("Src replica vanished for token %s, re-pick (attempt %d)",
+                            token, attempt)
+                continue
+            raise
+
+    assert migration_count == num_ops, f"Only completed {migration_count}/{num_ops} migrations"
+    logger.info("Completed tablet migration ops: %d/%d", migration_count, num_ops)
+
+
+def powers_of_two_in_range(lo: int, hi: int):
```

Comment:
This new test seems to cover all the cases, should we drop test_lwt_during_tablets_migration.py and test_lwt_during_tablets_resize.py? There is code/logic duplication between them and this file.